tools/rdm/TestMixins.py

```diff
@@ -77,6 +95,9 @@ class GetStringMixin(GetMixin):
     This mixin also sets a property if PROVIDES is defined.  The target class
     needs to defined EXPECTED_FIELD and optionally PROVIDES.
   """
+  MIN_LENGTH = 0
+  MAX_LENGTH = RDM_MAX_STRING_LENGTH
+
   def VerifyResult(self, response, fields):
     if response.WasAcked() and self.PROVIDES:
```

Comment:
Add 

string_field = fields[self.EXPECTED_FIELD]

so we can avoid repeating fields[self.EXPECTED_FIELD] so many times.